frontend/pages/hosts/details/HostQueryReport/HostQueryReport.tsx

```diff
@@ -131,7 +132,7 @@ const HostQueryReport = ({
             iconStroke
           >
             <>
-              View full query report
+              View data for all hosts
```

Comment:
Further iteration: Since we made the new query button have a breadcrumb that goes correctly back to the host details, I wonder if we can make the breadcrumb after clicking this return to this host query report instead of the queries page.

Not designed but would match the design change designed and deved here for adding a new query.

cc: @rachaelshaw